demos/object_detection_demo/python/object_detection_demo.py

```diff
@@ -91,6 +91,18 @@ def build_argparser():
     io_args.add_argument('-u', '--utilization_monitors', default='', type=str,
                          help='Optional. List of monitors to show initially.')
 
+    onnx_args = parser.add_argument_group('Onnx format options')
+    onnx_args.add_argument('--reverse_input_channels', default=False, action='store_true',
+                         help='Optional. Switch the input channels order from '
+                              'BGR to RGB for onnx model.')
+    onnx_args.add_argument('--mean_values', default=(0.0, 0.0, 0.0), type=float, nargs=3,
+                         help='Optional. Normalize input by subtracting the mean values '
+                              'per channel for onnx model. Example: 255 255 255')
+    onnx_args.add_argument('--scale_values', default=(1.0, 1.0, 1.0), type=float, nargs=3,
+                         help='Optional. Divide input by scale values per channel for onnx model. '
+                              'Division is applied after mean values '
+                              'subtraction. Example: 255 255 255')
```

Comment:
I think the references to ONNX can now be removed, since the options will work for any type of model.